Make text in the login greeter box easier to read on bright backgrounds.
by making the greeter box background more opaque.

diff --git a/src/dash-box.vala b/src/dash-box.vala
index 889ba41..b91886c 100644
--- a/src/dash-box.vala
+++ b/src/dash-box.vala
@@ -216,10 +216,10 @@ public class DashBox : Gtk.Box
 
         CairoUtils.rounded_rectangle (c, 0, box_y, box_w, box_h, box_r);
 
-        c.set_source_rgba (0.1, 0.1, 0.1, 0.4);
+        c.set_source_rgba (0.1, 0.1, 0.1, 0.9);
         c.fill_preserve ();
 
-        c.set_source_rgba (0.4, 0.4, 0.4, 0.4);
+        c.set_source_rgba (0.4, 0.4, 0.4, 0.9);
         c.set_line_width (1);
         c.stroke ();
 
